client/src/App.css

```diff
@@ -1 +1,7 @@
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: #f1f1f1}
 
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {
+  display: block;
+}
```

Comment:
Tailwind provides a way to achieve this as well https://tailwindcss.com/docs/hover-focus-and-other-states#group-hover. And kindly remove the `dropdown` and `dropdown-content` classes since there will be no need for them anymore

Reply:
So I went through tailwind and their hover state does not have `display-block`. I can add color to it and all but I am trying to display on hover and tailwind does not do that